scripts/replace-envvars.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
```

Comment:
I like `set -eux -o pipefail` in my bash scripts to:

* get feedback of what's being execute
* exit as soon as something errors
* error on undefined variables